packages/enzyme-test-suite/test/ShallowWrapper-spec.jsx

```diff
@@ -3527,6 +3459,13 @@ describe('shallow', () => {
             { foo: 'bar' }, { foo: 'baz' },
             { foo: 'context' },
           ]);
+        } else {
+          expected.push([
+            'componentDidUpdate',
+            { foo: 'props' }, { foo: 'props' },
+            { foo: 'bar' }, { foo: 'baz' },
+            undefined,
```

Comment:
I do not have enough insight into enzyme to evaluate whether this is acceptable, it smells bad. I wanted to create this PR anyway so you can guide me on how this should behave and fix it (or not), the code provoking this is in ShallowWrapper.js:413. If this is as weird to you as to me, I can give it more time and research how the adapter works, that option is undefined for React16.

Reply:
This will be fixed at #1261

Reply:
Will revert this change then.